components/DeleteModal.tsx

```diff
@@ -47,32 +42,56 @@ export function DeleteModal({
       <p className="row-start-2 row-end-2 ps-5 pt-5 text-lg w-full h-full font-bold">
         {dataToDeleteInfo}
```

Comment:
Selkeämpi ilman muuttujaa.
```suggestion
        {gift.name} - {gift.gift}
```